spec.md

```diff
@@ -443,7 +443,7 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| name* | string | A CLI-friendly name of the service. MUST only contain alphanumeric characters, periods, and hyphens (no spaces). MUST be unique across all service objects returned in this response. MUST be a non-empty string. |
+| name* | string | A CLI-friendly name of the service. MUST be unique across all service objects returned in this response. MUST be a non-empty string. |
```

Comment:
We have a bit of an inconsistency here because the definition of CLI-friendly say:
`... MUST only use alphanumeric characters, periods, and hyphens, with no spaces`
I'm assuming, based on the proposed change, that we should probably drop the CLI-friendly part of this. Or, perhaps say it SHOULD be a CLI-friendly name.

Reply:
Agreed - we'll drop the "CLI-friendly" from the first sentence.